index.html

```diff
@@ -4,7 +4,7 @@
     <link href="style.css" rel="stylesheet"/>
     <link rel="icon" href="images/favicon.ico"/>
     <meta charset="utf-8"/>
-    <meta name="keywords" content="bolides, nicholas costa, erik boesen, infinite games, html game, html 5, canvas game" />
+    <meta name="keywords" content="bolides, nicholas costa, erik boesen, infinite games, html game, html5, canvas game" />
```

Comment:
The `keywords` meta tag is not used by search engines anymore. You can safely remove it. In some cases, it even hurts page ranking.